mkt/tvplace/tests/test_serializers.py

```diff
@@ -0,0 +1,91 @@
+from django.contrib.auth.models import AnonymousUser
+from django.test.client import RequestFactory
+
+from nose.tools import ok_
+
+from mkt.site.tests import ESTestCase, TestCase, app_factory
+from mkt.tags.models import Tag
+from mkt.tvplace.serializers import (TVAppSerializer, TVESAppSerializer,
+                                     TVWebsiteSerializer,
+                                     TVESWebsiteSerializer)
+from mkt.webapps.indexers import WebappIndexer
+from mkt.websites.indexers import WebsiteIndexer
+from mkt.websites.utils import website_factory
+
+
+class TestAppSerializer(TestCase):
+
+    def refresh(self):
+        pass
+
+    def setUp(self):
+        self.creation_date = self.days_ago(1)
+        self.app = app_factory()
+        self.request = RequestFactory().get('/')
+        self.request.user = AnonymousUser()
+
+    def serialize(self, app, profile=None):
+        a = TVAppSerializer(instance=app, context={'request': self.request})
+        return a.data
+
+    def test_no_tv_featured(self):
+        res = self.serialize(self.app)
+        ok_(not res['tv_featured'])
+
+    def test_tv_featured(self):
+        Tag(tag_text='featured-tv').save_tag(self.app)
+        self.app.save()
+        self.refresh()
```

Comment:
I wonder if it would be a big perf hit on tests to just let it refresh all known indices... Worth testing someday to avoid further complications and simplify our tests.